src/RestaurantButton.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const RestaurantButton = props => {
+  return (
+    <button onClick={props.order} className="btn btn-primary m-2">
```

Comment:
Do you think that `order` is a good name for the prop here? Does it tell you what the prop does?